### PR TITLE
create isssue_template

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

**Comment by itkg-nanne** [comment]
:+1:

File: ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Informations
+<!--- Orchestra version (1.0, 1.1, master, ...) --->
+**Orchestra Version** : 
+
+**Issue Type**: 
+<!--- 
+Add x in type of your issue 
+For instance if it is an issue for a bug : 
+- [ x] Bug
+- [ ] Feature
```
Comment (itkg-ngilain) [inline ISSUE_TEMPLATE]: Maybe ``Feature request`` or even ``New feature request`` to be more explicit

File: ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] Question
+- [ ] Other
```
Comment (itkg-ngilain) [inline ISSUE_TEMPLATE]: What is the difference between ``Question`` and ``Other`` ?

File: ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+--->
+- [ ] Bug
+- [ ] Feature
+- [ ] Question
+- [ ] Other
+
+## Description
+<!--- 
+If is a bug issue
```
Comment (itkg-ngilain) [inline ISSUE_TEMPLATE]: If this is a

File: ISSUE_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Explain the issue with screenshot if possible and what do you expected
```
Comment (itkg-ngilain) [inline ISSUE_TEMPLATE]: I'll add ``Don't omit specific context such as brwoser version if related to a javascript/css bug for instance.``

**Comment by alavieille** [comment]
@itkg-ngilain

File: ISSUE_TEMPLATE [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] Feature request
+- [ ] Question
+--->
+- [ ] Bug
+- [ ] Feature request
+- [ ] Question
+
+## Description
+<!--- 
+If this is a a bug issue
```
Comment (itkg-ngilain) [inline ISSUE_TEMPLATE]: is a bug

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**